test/sampling.jl

```diff
@@ -170,6 +170,13 @@ check_sample_norep(a, (3, 12), 0; ordered=false)
 a = sample(3:12, 5; replace=false, ordered=true)
 check_sample_norep(a, (3, 12), 0; ordered=true)
 
+# tests of multidimensional sampling
+
+a = sample(1:4, (2, 2); replace=false)
+@test sort(vec(a)) == collect(1:4)
```

Comment:
Could you use `check_sample_norep` here as above?

Reply:
what is `check_sample_norep` supposed to do?

Reply:
AFAIK it just does a few consistency checks. That allows testing general cases even if the result is random (by definition).

Reply:
Maybe I'm not understanding how I have to use it, I'm getting this when I run it locally:
````
julia> a = sample(1:4, (2, 2); replace=false)
2×2 Array{Int64,2}:
 4  2
 3  1

julia> check_sample_norep(a, (1, 4), 5.0e-3; ordered=false)
Test Failed at test/sampling.jl:118
  Expression: isapprox(proportions(bj, vmin:vmax), p0, atol=ptol)
   Evaluated: isapprox([0.0, 0.5, 0.0, 0.5], [0.25, 0.25, 0.25, 0.25]; atol=0.005)
ERROR: There was an error during testing
````

Reply:
Replace `5.0e-3` with `0`?

Reply:
alright, done